code/game/jobs/jobs.dm

```diff
@@ -44,6 +44,7 @@ var/const/ASSISTANT			=(1<<11)
 var/const/BRIDGE			=(1<<12)
 var/const/CLOWN				=(1<<13) //VOREStation Add
 var/const/MIME				=(1<<14) //VOREStation Add
+var/const/INTERN			=(1<<15) //VOREStation Add

```

Comment:
If INTERN is a rename of assistant (/datum/job/assistant) why do we need a new constant instead of the ASSISTANT constant?

Reply:
Fixed